src/main/java/appeng/integration/IntegrationNode.java

```diff
@@ -98,10 +94,18 @@ void call( final IntegrationStage stage )
 
 						if( enabled )
 						{
-							this.classValue = this.getClass().getClassLoader().loadClass( this.name );
-							this.mod = (IIntegrationModule) this.classValue.getConstructor().newInstance();
-							final Field f = this.classValue.getField( "instance" );
-							f.set( this.classValue, this.setInstance( this.mod ) );
+							switch( type )
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a default case to this switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)